src/maintainer/knowledge_base.rst

```diff
@@ -791,6 +793,52 @@ Following implies that ``python`` is a runtime dependency and a Python matrix fo
 
 You need to rerender the feedstock after this change.
 
+Using CMake
+===========
+
+`CMake <https://cmake.org/>`__ can be used to build more complex projects in ``build.sh``
+or ``bld.bat`` scripts.
+
+If you are using cmake, be sure to make it a build requirement in the ``build`` or ``host`` section.
+
+.. code-block:: yaml
+
+    build:
+      - cmake
+
+CMake requires you to supply a configuration file, usually called ``CMakeLists.txt``.
+You can use the one from your source tar ball if it exists or include your own in the
+same directory as the conda-forge recipe file and copy it to the source directory.
+
+You can tell CMake which Python to use by passing ``-DPython3_EXECUTABLE="$PYTHON"``
+(macOS or Linux) or ``-DPython3_EXECUTABLE="%PYTHON%"`` (Windows) as a command line option
+to CMake.
+
+Here are some basic commands to get you started. These are highly dependent on your source
+code layout and aren't intended to be used "as is".
+
+.. code-block::
+
+    cmake CMakeLists.txt -DPython3_EXECUTABLE="$PYTHON"
+    cmake --build . --config Release
+
+.. code-block::
+
+    cmake -G "NMake Makefiles" -DCMAKE_BUILD_TYPE:STRING=Release -DPython3_EXECUTABLE="%PYTHON%"
```

Comment:
The `-G "NMake Makefiles"` is Windows specific. Should we put that bit in the Windows CMake section?

Reply:
It's already there. The reason I left it is that I didn't want to write a self contained example for Mac/Linux (just time constraints). And it seemed strange to have only the Mac/Linux example at that point. I'll think about that one.

Reply:
I left it in the latest revision, but added a note to refer to the other section too. Does this work for you?

Reply:
Yeah that seems fine. Thanks!